src/test/java/edu/ksu/canvas/attendance/services/CanvasApiWrapperServiceUTest.java

```diff
@@ -1,14 +1,19 @@
 package edu.ksu.canvas.attendance.services;
 
 import edu.ksu.canvas.CanvasApiFactory;
+import edu.ksu.canvas.interfaces.AssignmentOverrideWriter;
 import edu.ksu.canvas.interfaces.EnrollmentReader;
 import edu.ksu.canvas.model.Enrollment;
 import edu.ksu.canvas.model.Section;
+import edu.ksu.canvas.model.assignment.Assignment;
+import edu.ksu.canvas.model.assignment.AssignmentOverride;
 import edu.ksu.canvas.requestOptions.GetEnrollmentOptions;
 import edu.ksu.lti.launch.exception.NoLtiSessionException;
 import edu.ksu.lti.launch.model.LtiSession;
 import edu.ksu.canvas.oauth.OauthToken;
 import edu.ksu.lti.launch.service.LtiSessionService;
+import org.apache.log4j.Logger;
+import org.hamcrest.Matchers;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.hamcrest.Matchers'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AUselessImportCheck)